bitnami/clickhouse/Chart.yaml

```
@@ -34,4 +34,4 @@ maintainers:
 name: clickhouse
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/clickhouse
-version: 9.3.0
+version: 9.4.0
```

Please bump patch version instead:
```suggestion
version: 9.3.1
```